src/config.js

```diff
@@ -67,7 +67,7 @@ const config = convict({
     screenshotsBucket: {
       doc: 'Bucket containing screenshots',
       format: String,
-      default: 'argos-screenshots-dev',
+      default: 'argos-screenshots-dev-2',
```

Comment:
I have deleted the bucket to move it to US-standard from Irland location. I had to change the name.